Lombiq.Tests.UI/Services/OrchardCoreInstance.cs

```diff
@@ -104,14 +109,14 @@ public async Task<Uri> StartUpAsync()
                 .CopyAppConfigFiles(Path.GetDirectoryName(_configuration.AppAssemblyPath), _contentRootPath);
         }
 
-        // If you try to use dotnet.exe to run a DLL published for a different platform (seen this with running
+        // If you try to use the dotnet command to run a DLL published for a different platform (seen this with running
         // win10-x86 DLLs on an x64 Windows machine) then you'll get a "Failed to load the dll from hostpolicy.dll
         // HRESULT: 0x800700C1" error even if the exe will run without issues. So, if an exe exists, we'll run that.
```

Comment:
Let's be consistent with DLL for improved readability:
```suggestion
        // If you try to use the dotnet command to run a DLL published for a different platform (seen this with running
        // win10-x86 DLLs on an x64 Windows machine) then you'll get a "Failed to load the dll from hostpolicy.dll
        // HRESULT: 0x800700C1" error even if the EXE will run without issues. So, if an EXE exists, we'll run that.
```

Reply:
But not too consistent.
![image](https://user-images.githubusercontent.com/4007293/170149292-e769a223-5e02-4568-a5f1-14a0a4e415ec.png)

I think we never capitalize extensions anywhere else so probably that "DLLs" is what should be replaced.

Reply:
I didn't replace that one because it's inside an error message 😁

Reply:
I find dll and exe hard to read in the middle of a sentence to be honest.

Reply:
Agreed, but we shouldn't break convention just here.

Reply:
Yeah, that's the hardest part. The go ahead and make it consistent in some way, at least.